libats/src/main/scala/com/advancedtelematic/libats/http/ErrorHandler.scala

```diff
@@ -68,18 +72,40 @@ object Errors {
     ).foldLeft(PartialFunction.empty[Throwable, Route])(_ orElse _)
 }
 
+trait DeployEnvSupport {
+  protected lazy val isProd =
+    Option(System.getenv("DEPLOY_ENV")).exists(env => env == "production" || env.isEmpty)
```

Comment:
Do you plan on adding this environment variable? I don't see it in marathon.

Reply:
Will add it after this is merged, waiting for production to be deployed using the new scripts